basilica/basilica/__init__.py

```diff
@@ -197,7 +202,7 @@ def embed_image_files(self, image_files, model='generic', version='default',
         def load_image_files(image_files):
             for image_file in image_files:
                 with open(image_file, 'rb') as f:
-                    yield f.read()
+                    yield self.__resize_image(f.read())
```

Comment:
I don't think we need to do this here, it's already being done in `embed_images`.